frontend-instructor/src/components/navBar/NavBar.jsx

```
@@ -0,0 +1,9 @@
+import DesktopNav from "./DesktopNav";
+
+export default function NavBar() {
+  return (
+    <>
+      <DesktopNav></DesktopNav>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)